pkg/cmd/create/edge_storage/bucket.go

```diff
@@ -0,0 +1,99 @@
+package edge_storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"go.uber.org/zap"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	sdk "github.com/aziontech/azionapi-go-sdk/storage"
+
+	msg "github.com/aziontech/azion-cli/messages/edge_storage"
+	api "github.com/aziontech/azion-cli/pkg/api/storage"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/utils"
+)
+
+type Fields struct {
+	Name       string
+	EdgeAccess string
+	FileJSON   string
+}
+
+func NewBucket(f *cmdutil.Factory) *cobra.Command {
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.USAGE_BUCKET,
+		Short:         msg.SHORT_DESCRIPTION_CREATE_BUCKET,
+		Long:          msg.LONG_DESCRIPTION_CREATE_BUCKET,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example:       heredoc.Doc(msg.EXAMPLE_CREATE_BUCKET),
+		RunE: runE(f, fields),
+	}
+
+	flags := cmd.Flags()
+	addFlags(flags, fields)
+	return cmd
+}
+
+func runE(f *cmdutil.Factory, fields *Fields) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		request := api.RequestBucket{}
+		if cmd.Flags().Changed("file") {
+			err := utils.FlagFileUnmarshalJSON(fields.FileJSON, &request)
+			if err != nil {
+				return utils.ErrorUnmarshalReader
+			}
+		} else {
+			err := createRequestFromFlags(cmd, fields, &request)
+			if err != nil {
+				return err
+			}
+		}
+
+		client := api.NewClient(f.HttpClient, f.Config.GetString("storage_url"), f.Config.GetString("token"))
+		err := client.CreateBucket(context.Background(), request)
+		if err != nil {
+			return fmt.Errorf(msg.ERROR_CREATE_BUCKET, err)
+		}
+
+		logger.LogSuccess(f.IOStreams.Out, msg.SUCCESS_CREATE_BUCKET)
+		return nil
+	}
+}
+
+func createRequestFromFlags(cmd *cobra.Command, fields *Fields, request *api.RequestBucket) error {
+	if !cmd.Flags().Changed("name") {
+		answers, err := utils.AskInput(msg.ASK_NAME_CREATE_BUCKET)
+		if err != nil {
+			logger.Debug("Error while parsing answer", zap.Error(err))
+			return utils.ErrorParseResponse
+		}
+		fields.Name = answers
+	}
+	if !cmd.Flags().Changed("edge-access") {
+		answers, err := utils.AskInput(msg.ASK_EDGE_ACCESSS_CREATE_BUCKET)	
```

Comment:
If there is a set of values accepted, I think it would be better to present a list of possible values rather than ask for them to write the whole thing